std_msgs/Header header
telecoV/Waypoint[] waypoints